e2e/core-e2e/jest.config.js

```diff
@@ -11,5 +11,5 @@ module.exports = {
   },
   moduleFileExtensions: ['ts', 'js', 'html'],
   coverageDirectory: '../../coverage/e2ecore-e2e',
-  testTimeout: 60000,
```

Comment:
Jest is a bit slower in v27 for nx plugins for some reason, we experienced this on the nrwl e2e as well